distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -833,23 +849,112 @@ public void testCswQueryByTitle() throws Exception {
 
     @Test
     public void testCswQueryForMetacardXml() throws Exception {
-        String titleQuery = getCswQuery("title",
-                "myTitle",
-                "application/xml",
-                "urn:catalog:metacard");
 
-        given().contentType(ContentType.XML)
-                .body(titleQuery)
-                .when()
-                .post(CSW_PATH.getUrl())
-                .then()
-                .assertThat()
-                .body(hasXPath("/GetRecordsResponse/SearchResults/metacard/@id",
-                        is(metacardIds[GEOJSON_RECORD_INDEX])),
-                        hasXPath("/GetRecordsResponse/SearchResults/@numberOfRecordsReturned",
-                                is("1")),
-                        hasXPath("/GetRecordsResponse/SearchResults/@recordSchema",
-                                is("urn:catalog:metacard")));
+        // The first few bytes of the metacard's base-64 encoded thumbnail image
```

Comment:
This PR adds tests the 3 output schemas of the CSW endpoint. I basically co-opted the `testCswQueryForMetacardXml` to test the native DDF Metacard output schema. The summary of changes to this method are:
1. Added additional assertions to verify the record returned by the CSW federated query.
2. Extract the section of the xml that represents the metacard using a regex.
3. Get the input transformer service for metacard XML. 
4. Transform the XML into a metcard object and assert a couple of properties to ensure some basic data is present. The intention was not to test EVERY attribute, just a couple.